### PR TITLE
Reduce bundle size

### LINKED ISSUE #7286: Build Open MCT with Terser plugin to reduce library size.
**Is your feature request related to a problem? Please describe.**
Experimentation has shown that the Terser plugin may significantly reduce the size of our built library.

We should also consider using Terser in some of our other projects (viper-openmct, openmct-yamcs, openmct-mmgis).

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7286 

### Describe your changes:

- Use minified plotly
- Use ESM imports for the d3 libraries
- Use ESM imports for printj

Brings unpacked bundle size from 26.3MB - 24.3MB
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: .webpack/webpack.common.js [hunk 1]
```diff
@@ -71,8 +71,7 @@ const config = {
       bourbon: 'bourbon.scss',
       'plotly-basic': 'plotly.js-basic-dist',
       'plotly-gl2d': 'plotly.js-gl2d-dist',
-      'd3-scale': path.join(projectRootDir, 'node_modules/d3-scale/dist/d3-scale.min.js'),
-      printj: path.join(projectRootDir, 'node_modules/printj/dist/printj.min.js'),
+      printj: 'printj/printj.mjs',
```
Comment (evenstensberg) [inline .webpack/webpack.common.js:74]: Would it make sense to merge #7038 first?

Comment (ozyx) [reply .webpack/webpack.common.js:74]: Yes, I'll definitely be taking another look at your PR this week as I'm diving into our build/packaging process. I may split the webpack changes out of this PR, which is kind of experimental at this point anyway. Thanks for your patience again, really appreciate it.

Comment (evenstensberg) [reply .webpack/webpack.common.js:74]: Np 🤗

**Comment by davetsay** [comment]
👍 for issue name and reducing the bundle size

**Comment by scottbell** [comment]
@ozyx what's the best way to test this?

**Comment by ozyx** [comment]
> @ozyx what's the best way to test this?

Something like:

1. Clone a fresh copy of openmct
2. git checkout master
3. npm i; npm pack
4. Move the generated *.tgz archive to another folder (outside of openmct repo)
5. Repeat the same but on this branch 
6. compare packed / unpacked space
7. Manual smoke test of basic usage

I thought we could compress even smaller but I think best we can do is a few MB, without removing e2e or source files/maps (which I don't think we want to do). As a whole our package size has been on a downward trend since 1.8 which is good!

File: src/api/telemetry/TelemetryValueFormatter.js [hunk 2]
```diff
@@ -20,133 +20,132 @@
...
-        return value.replace(arrayRegex, '');
-      }
-
-      return value;
+import _ from 'lodash';
+import { sprintf } from 'printj';
+
+// TODO: needs reference to formatService;
+export default function TelemetryValueFormatter(valueMetadata, formatMap) {
+  const numberFormatter = {
```
Comment (evenstensberg) [inline src/api/telemetry/TelemetryValueFormatter.js:28]: maybe as inline object method?


`parse(x) {...}`

**Review by scottbell — APPROVED** [review]
I think changing `TelemetryValueFormatter` to an es6 class would be cool, but not necessary for this PR.

File size was reduced:
```
-rw-r--r--@  1 scott  staff   5.4M Dec 19 12:11 openmct-3.3.0-honey.tgz
-rw-r--r--@  1 scott  staff   5.9M Dec 19 12:07 openmct-3.3.0-master.tgz
```
nice job! Smoke test of functionality went well.

File: src/api/telemetry/TelemetryValueFormatter.js (hunk 2 above)
Comment (scottbell) [reply src/api/telemetry/TelemetryValueFormatter.js]: gorgeous

**Pull request merged**